src/api/app/assets/javascripts/webui/application/comment.js

```diff
@@ -22,3 +22,15 @@ function setup_comment_page() {
         $(this).find('input[type="submit"]').prop('disabled', true);
     });
 }
+
+$(document).ready(function(){
+    $('a.delete_link').on('ajax:success', function(e, data, status, xhr){
```

Comment:
Can you please rename `e` in a speaking variable, please? Even if it's not used.